client/src/main/java/io/confluent/kafka/schemaregistry/client/CachedSchemaRegistryClient.java

```diff
@@ -795,11 +866,11 @@ public String toString() {
     }
   }
 
-  static class SubjectAndId {
+  static class SubjectAndInt {
```

Comment:
why renaming here? Looks like SubjectAndId is more straightforward.

Reply:
This is now used for both subject and id, as well as subject and version